code-challenges.test.js

```diff
@@ -13,44 +13,122 @@
 
 // --------------------1) Create a function that takes in a number (greater than 2) and returns an array that length containing the numbers of the Fibonacci sequence.
 
-// a) Create a test with expect statements for each of the variables provided.
-
 // Example input: 6
 // Expected output: [1, 1, 2, 3, 5, 8]
 
 // Example input: 10
 // Expected output: [1, 1, 2, 3, 5, 8, 13, 21, 34, 55]
 
+// a) Create a test with expect statements for each of the variables provided.
+
+// a describe method that list the name of the function or naming of the particulat test
+describe("fibNum", () => {
+    it("the sequence in which each number is the sum of the two preceding ", () => {
+        expect(fibonacci(6)).toEqual(1, 1, 2, 3, 5, 8)
+    })
+})
+
+
+// Pseudo code:
+// 1. This function takes a input number greater than 2
+// 2. Declare 2 variables that takes tracks the previous and current numbers
+// 3. - 2 variables, add the 2 variables
+// 4. - create an array that shows the sum of the 2 variables
+
 
 // b) Create the function that makes the test pass.
 
+var fibNum(num) => {
+    if ((fibNUM) > = 2 {
+        return `number needs to be greater than 2`
+    }
+    if else (input > 2) {
+        let num1 = 1,
+            num2 = 2
 
-// --------------------2) Create a function that takes in an array and returns a new array of only odd numbers sorted from least to greatest.
+        let array = [num1, num2]
+        for (let i = 0; i < input.length - 2, i++) {
+            let num = num1 + num2
+            array.push(sum)
+            num1 = array[2 + i]
+            num1 = array[1 + i]
+        }
+        return array
+    }
```

Comment:
Very close. I see just a few syntax errors to work through. On line 42 I think you are wanting to create an evaluation about your argument `num`. Maybe something like `if(num >= 2)`